src/chesster.js

```diff
@@ -890,7 +898,7 @@ function validateUserResult(details, result){
 
 function processGamelink(bot, message, gamelink, options, userResult){
     //get the gamelink id if one is in the message
-    var result = spreadsheets.parseGamelink(gamelink);
+    var result = spreadsheet.parseGamelink(gamelink);
```

Comment:
whats happening here? related to my other question about spreadsheet(s)

maybe we have 2 for a reason. or our we just fumbling two variables at the moment?